articles/quickstart/spa/react/01-login.md

```diff
@@ -52,9 +52,7 @@ The `Auth0Provider` component takes the following props:
 
 - `domain` and `clientId`: The values of these properties correspond to the "Domain" and "Client ID" values present under the "Settings" of the single-page application that you registered with Auth0.
 
-:::note
-If you are using a [custom domain with Auth0](https://auth0.com/docs/custom-domains), the value of the domain prop is the value of your custom domain instead of the value reflected in the "Settings" tab.
-:::
+<%= include('../_includes/_auth_note_custom_domains') %>
```

Comment:
Ah. Beware of my "prop" comment at the beginning, as you're reusing this with React as well.

Reply:
I'll try to make this a bit more generic.

Reply:
Oh, I had already changed this when I made the include:

> If you are using a [custom domain with Auth0](/custom-domains), the value of the domain property is the value of your custom domain instead of the value reflected in the "Settings" tab.

Reply:
That looks good 👍